dev-lua/luv/luv-1.30.1_p0.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MIN_VERSION="2.8"
```

Comment:
The lowest in tree is 3.13. This line can be removed.

Reply:
Done :)